client/README.md

```diff
@@ -5,6 +5,8 @@
 [![Python](https://img.shields.io/badge/Python-3.8%20%7C%203.9%20%7C%203.10-informational)](https://www.python.org/)
 [![Qiskit](https://img.shields.io/badge/Qiskit-%E2%89%A5%200.39.0-6133BD)](https://github.com/Qiskit/qiskit)
 
+# IMPORTANT: THIS PACKAGE WAS DEPRECATED IN FAVOR OF QISKIT-SERVERLESS PACKAGE
+
```

Comment:
let's add link here to new location of repo

Reply:
Done [f4857a4](https://github.com/Qiskit/qiskit-serverless/pull/1326/commits/f4857a4764b91296117842ffb10bbf4d6999aa4b)